lib/engine/game/g_1856/step/dividend.rb

```diff
@@ -75,7 +43,11 @@ def holder_for_corporation(_entity)
             @game.share_pool
           end
 
-          def log_run_payout(entity, kind, revenue, action, payout)
+          def total_revenue
+            super - (@round.interest_penalty[@round.current_operator] || 0)
```

Comment:
I think the previous construct might be a bit more clearer

            penalty = @round.interest_penalty[entity] || 0
            revenue = @game.routes_revenue(routes) - penalty
            
            i.e. to put the @round.in.... into penalty.

Reply:
penalty is actually a bad term as its not a penalty, but more so garnishing of revenue.